tools/banner-studio/index.html

```
@@ -136,6 +147,9 @@
   <label>Corner radius <span id="rdV"></span></label><input type="range" id="radius" min="0" max="0.09" step="0.002">
   <label>Brand glow <span id="gwV"></span></label><input type="range" id="glow" min="0" max="0.16" step="0.005">
   <label>Vignette <span id="vgV"></span></label><input type="range" id="vignette" min="0" max="1" step="0.02">
+  <label>Gradient type</label><select id="gradType"><option value="radial">Radial</option><option value="linear">Linear</option></select>
+  <label>Gradient angle <span id="gaV"></span></label><input type="range" id="gradAngle" min="0" max="360" step="5">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)